src/altinn-app-frontend/src/components/index.ts

```diff
@@ -161,20 +163,25 @@ export const schemaComponents: IComponent[] = [
   },
   {
     name: 'NavigationBar',
-    Tag: NavigationBar,
+    Tag: NavigationBarComponent,
     Type: ComponentTypes.NavigationBar,
   },
   {
     name: 'Likert',
     Tag: LikertComponent,
     Type: ComponentTypes.Likert,
   },
+  {
+    name: 'Panel',
+    Tag: PanelComponent,
+    Type: ComponentTypes.Panel,
+  },
 ];
 
 export const advancedComponents: IComponent[] = [
   {
     name: 'AddressComponent',
-    Tag: AddressComponent,
+    Tag: Address,
```

Comment:
This change seems to be the opposite of the naming convention for all the other components here - is there a reason for this?

Reply:
It was mainly to get rid of some [reported code smells from sonarcloud](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&id=Altinn_app-frontend-react&open=AX_0PbP8ruzTKI3lOPDv), although I think the report is showing a false positive here. It mentions that `AddressComponent` is already declared, but that shouldnt matter since its delared within the enum, so seems like it doesnt handle enums very well?

Even though the original code was not problematic after all, I though it would be good to get rid of the reported code smells just to reduce noise. The name here is not exposed anywhere as far as I can tell so it shouldnt cause any problems.